src/packages/aws-cognito/src/util.ts

```diff
@@ -47,7 +47,7 @@ export const createUser = async (
 	if (!user) return null;
 	return {
 		...user,
-		Groups: groups.split(','),
+		Groups: groups?.split(','),
```

Comment:
Might be a bit nicer to do:
```suggestion
		Groups: groups?.split(',') ?? [],
```